api/src/pcapi/routes/backoffice/templates/venue/get.html

```diff
@@ -323,12 +323,26 @@ <h5 class="modal-title">Supprimer le lien entre le lieu {{ venue.common_name }}
               <p class="mb-1">
                 <a href="{{ venue | pc_pro_venue_bookings_link }}"
                    target="_blank"
-                   class="fw-bold link-primary">Réservations</a>
+                   class="fw-bold link-primary">
+                  Réservations
+                  <i class="bi bi-box-arrow-up-right"></i>
+                </a>
               </p>
               <p class="mb-1">
                 <a href="{{ venue | pc_pro_venue_offers_link }}"
                    target="_blank"
-                   class="fw-bold link-primary">Offres associées</a>
+                   class="fw-bold link-primary">
+                  Offres associées
+                  <i class="bi bi-box-arrow-up-right"></i>
+                </a>
+              </p>
+              <p class="mb-1">
+                <a href="{{ url_for("backoffice_web.offer.list_offers") }}?search-0-search_field=VENUE&search-0-venue={{ venue.id }}&search-0-operator=IN"
```

Comment:
Passer les variables à `url-for` ne passe en effet pas avec les `-`, mais tu peux contourner en passant les kwargs dans un dictionnaire :
```suggestion
                <a href="{{ url_for("backoffice_web.offer.list_offers", **{"search-0-search_field": "VENUE", "search-0-operator": "IN", "search-0-venue": venue.id}) }}"
```